org.kde.dolphin.json

```diff
@@ -9,20 +9,25 @@
         "--filesystem=/media",
         "--filesystem=/run/media",
         "--filesystem=home",
+        "--own-name=org.kde.dolphin-2",
```

Comment:
That looks weird

Reply:
This looks like the same issue as with Okular: https://github.com/flathub/org.kde.okular/pull/375

Reply:
Ah interesting. Seems I don't know enough about dbus still...

Reply:
Pushed a fix.